src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/PropertySymbol.cs

```diff
@@ -31,5 +31,7 @@ public IndexerSymbol AsIndexerSymbol()
             Debug.Assert(isIndexer());
             return (IndexerSymbol)this;
         }
+
+        public bool Bogus { get; set; }
```

Comment:
`isIndexer` and `AsIndexerSymbol` should ideally have been changed when getting rid of the similar methods defined elsewhere, anyway.